templates/navigation.hbs

```diff
@@ -1,6 +1,9 @@
 {{! Maintainers - Daryl Hedley}}
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link" aria-label="Skip navigation" data-event="skipNavigation" role="button">Skip Navigation</button>
```

Comment:
could you change line 5 to:
```
<button class="skip-nav-link" aria-label="{{_globals._accessibility._ariaLabels.skipNavigation}}" data-event="skipNavigation" role="button">Skip Navigation</button>
```

and add _globals._accessibility._ariaLabels.skipNavigation = "Skip Navigation" to the appropriate schema + course.json?